src/TestHelper/RequestObjectTestHelper.php

```diff
@@ -144,14 +144,19 @@ public function getRequestProperties(RequestObjectInterface $object): array
         $methodsToCheck = \array_filter(
             \array_diff($instanceMethods, $interfaceMethods),
             static function (string $method): bool {
-                return \strncmp($method, 'get', 3) === 0;
+                return \strncmp($method, 'get', 3) === 0 ||
+                    \strncmp($method, 'is', 2) === 0;
             }
         );
 
         $actual = [];
         foreach ($methodsToCheck as $method) {
-            $property = \lcfirst(\substr($method, 3));
+            $length = 3;
```

Comment:
Its better in this case to use a ternary rather than reassigning the variable.

`$length = \strncmp(...) === 0 ? 2 : 3;`

It also makes me wonder how we should handle priorities - should the methods be sorted alphabetically? Should we extract get* first then look for is*?

Reply:
Yea.. make sense to add ternary operator. 
It is now sorting alphabetically.

Reply:
My comment was a question - is alphabetical sorting correct?

should we prioritise get over is just because `get` comes before `is`?

What happens when the entity has both a isThing() and a getThing() and which one is more appropriate for the RequestObjectTestHelper to call?

Reply:
I think as you said initially, `get` should be prioritised over `is` as we are using `get` more on requests than `is`(rarely used).
In the above example, `getThing()` should be called first.
Even though, we aren't sorting it alphabetically it should work because our public method names are sorted alphabetically in the class as per standards.
`asort()` will take care of it in case someone forgot to turn on standards.

Reply:
Just to test whether all the `gets` are returned first, rearranged the `TestRequest` not to follow alphabetical order.